todo_app/app.py

```diff
@@ -1,14 +1,28 @@
-from flask import Flask
+from flask import Flask, render_template, request
+
 
 from todo_app.flask_config import Config
+from todo_app.data import session_items
+
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = session_items.get_items()
+    return render_template("index.html", items = items)
+
+@app.route('/newitem', methods=['POST'])
+def CreateNew():
+
+    #call add item
+    data = request.form.get('newitem')
+    session_items.add_item(data)
+    #re render
+    items = session_items.get_items()
+    return render_template("index.html", items = items)
```

Comment:
Instead of duplicating this code, you could use Flask's redirect function. `return redirect('/')`

**But if you are running in GitPod,** then redirect won't "just work" because the app thinks it's running on localhost:5000. I can explain a workaround if you'd like.